lib/evolution_table_query.js

```diff
@@ -5,6 +5,16 @@ const initTableQuery = `CREATE TABLE IF NOT EXISTS $EVOLUTIONS (
     primary key (id)
 );`;
 
+const initTableExistsQuery = `SELECT EXISTS (
```

Comment:
This query isn't engine-agnostic. E.g., for SQLite we should use something like this `SELECT name FROM sqlite_master WHERE type ='table' AND name='evolutions';`

Actually, it's the reason, why I've used `IF NOT EXISTS` instead of direct check.

If you have idea, how we can fix it — we can try to implement it.

Reply:
Well, if we are considering unified SQL script approach is it correct to say rdms list will be limited with postgre, sqllite and mysql-like-rdmss? E.g. MSSQL also will not be applicable in this case (it doesn't suppor decribed script structure).
As a brute force approach it can be case selection via `rdms` parameter.

Reply:
I don't understand, why we should fix something that isn't broken now 🤔

What problems do you find in `IF NOT EXISTS` query?

Reply:
It's not a fix, but enhancement. Me personally was confused a little bit by logs, observing evolution table is being created each and every time I'm running `yarn trona`. Avoiding unnecessary logs message looks logical and this 'smart' behavior addresses this issue. No begging - we can drop it.

Reply:
Maybe we can reduce a count of logs without breaking the core feature (engine  agnosticism)? 🌚

To be clear, I like the idea of reducing extra-logs, but I wouldn't like to lock this library on specific SQL-engines.

Reply:
Ok, I will try to adjust PR in order to keep agnosticism and reduce confusing log messages